pkg/features/features.go

```diff
@@ -67,6 +67,7 @@ const (
 	// owner: @ScheererJ @DockToFuture
 	// alpha: v1.22.0
 	// beta: v1.42.0
+	// deprecated: v1.61.0
```

Comment:
See version comment above.
Probably it is now also GA and not deprecated.

Reply:
fixed.